httpx/client.py

```diff
@@ -222,6 +223,19 @@ def params(self, params: QueryParamTypes) -> None:
         timeout: typing.Union[TimeoutTypes, UnsetType] = UNSET,
         trust_env: bool = None,
     ) -> Response:
+        if cert is not None:
+            warnings.warn(
+                "Passing a 'cert' argument when making a request on a client "
+                "is due to be deprecated. Instantiate a new client instead, "
+                "passing any 'cert' or 'verify' arugments to the client itself."
+            )
+        if verify is not None:
+            warnings.warn(
+                "Passing a 'verify' argument when making a request on a client "
+                "is due to be deprecated. Instantiate a new client instead, "
+                "passing any 'cert' or 'verify' arugments to the client itself."
```

Comment:
Typo:

```suggestion
                "passing any 'cert' or 'verify' arguments to the client itself."
```

Reply:
I really need to find a good process for making sure those aren't creeping in all the time!

Reply:
Time for `mkspellcheck`? 🤷‍♂